scripts/html-email/templates/oob_code_email.mjml

```diff
@@ -0,0 +1,26 @@
+<mjml>
```

Comment:
Any instructions on how to transpile this?

Reply:
As a comment in the new make target.